docs/data/routes/docs/nextjs/experience-editor/walkthrough/en.md

```diff
@@ -3,4 +3,109 @@ name: walkthrough
 routeTemplate: ./data/component-templates/article.yml
 title: Connect to the Experience Editor
 ---
-# Walkthrough: Connect to the Experience Editor
\ No newline at end of file
+# Walkthrough: Connect your Next.js app to the Experience Editor
+
+A Next.js application utilizes the [JSS Http rendering engine](/docs/fundamentals/services/view-engine) feature for integration to the Sitecore Experience Editor. This article will guide you through the required configuration.
+
+## Step 1: Sitecore configuration
+
+> A default Sitecore configuration file is included with the Next.js app upon creation with `jss create`. You can find this file under `/sitecore/config`.
+
+### JSS app configuration
+
+You must set the following attributes on the JSS `<app />`:
+
+* `serverSideRenderingEngine` - Must be set to `http`
+* `serverSideRenderingEngineEndpointUrl` - The absolute URL of your Next.js app's API route which handles POST requests from the Sitecore Experience Editor. By default, this API route is '/api/editing/render'.
+* `serverSideRenderingEngineApplicationUrl` - The root URL that will be used when creating absolute links in the rendered HTML. Link replacement is necessary in order for relative links in the rendered HTML to be able to load in the Experience Editor.
+
+> Learn more about JSS app configuration and the available attributes [here](/docs/fundamentals/services/app-configuration).
+
+For example, the default configuration will be:
+
+```xml
+<configuration xmlns:patch="http://www.sitecore.net/xmlconfig/">
+  <sitecore>
+    <javaScriptServices>
+      <apps>
+        <app name="exampleApp"
+            ...
+            serverSideRenderingEngine="http"
+            serverSideRenderingEngineEndpointUrl="http://localhost:3000/api/editing/render"
+            serverSideRenderingEngineApplicationUrl="http://localhost:3000"
+            ...
+        />
+      </apps>
+    </javaScriptServices>
+  </sitecore>
+</configuration>
+```
+
+### JSS Editing Secret
+
+To secure the Experience Editor endpoint exposed by your Next.js app (`serverSideRenderingEngineEndpointUrl` above), a secret token is used. This secret must be set on both server-side and client-side, and the values must match.
+
+> We recommend an alphanumeric value of at least 16 characters.
+
+You have 2 options for setting this on the server-side.
+
+**Option 1: Environment variable**
+
+Set the `SITECORE_JSS_EDITING_SECRET` environment variable. This could be preferred for container or hosted production environments (e.g. Vercel).
```

Comment:
Vercel wouldn't be applicable for the Sitecore instance, which is what we are talking about here, right?

Reply:
Good catch - correct. Had client-side .env on the brain.